### PR TITLE
Implement headteacher salaries (merge into existing Salaries PR)

### PR DESCRIPTION
**Pull request by MylesJarvis**
### Trello card

Removing Trello card as it appears to be failing on the pipeline.

### Context

We're centralising monetary values in one place.

### Changes proposed in this pull request

Pages with changes:
- https://get-into-teaching-app-review-4034.test.teacherservices.cloud/is-teaching-right-for-me/teacher-pay-and-benefits#headteacher-salary
- https://get-into-teaching-app-review-4034.test.teacherservices.cloud/landing/careers
- https://get-into-teaching-app-review-4034.test.teacherservices.cloud/landing/grow
- https://get-into-teaching-app-review-4034.test.teacherservices.cloud/landing/how-much-do-teachers-get-paid
- https://get-into-teaching-app-review-4034.test.teacherservices.cloud/landing/what-teaching-taught-me

- This merges into an existing branch.
- I've made a few of these, so there may be merge conflicts in the salaries.yml file (relating to the heading of 'Implemented below this line'.
- You'll need to ensure you're happy with the ordering of headings before you merge the main PR.

### Guidance to review

- Review the pages above and merge into the larger Salary PR once ready (I'd recommend doing that after you've checked and verified that specific PR again).
- https://github.com/DFE-Digital/get-into-teaching-app/pull/3977

### Pre-election period restrictions

* [ ] This change is permitted within the constraints of the [pre-election period guidelines](https://www.gov.uk/government/publications/election-guidance-for-civil-servants/general-election-guidance-2024-guidance-for-civil-servants-html#publishing-content-online-)

### CODE REVIEW

File: app/views/content/landing/careers/_content.html.erb [hunk 1]
```diff
@@ -41,7 +41,7 @@
       <ul>
       <li>most teachers get a pay rise every year, depending on your school. You could earn over £41k within 5 years</li>
       <li>you may get extra payments for taking on more responsibilities, for example creating a timetable or becoming head of a department</li>
-      <li>as a headteacher you can expect to earn between £53k to £139k depending on your location and school</li>
+      <li>as a headteacher you can expect to earn between <%= v :salaries_headteacher_minshortened %> to <%= v :salaries_headteacher_maxshortened %> depending on your location and school</li>
```
Comment (Sarah-DfE) [inline app/views/content/landing/careers/_content.html.erb:44]:
```suggestion
      <li>as a headteacher you can expect to earn between <%= v :salaries_headteacher_minshortened %> and <%= v :salaries_headteacher_maxshortened %> depending on your location and school</li>
```

File: app/views/content/landing/grow/_content.html.erb [hunk 2]
```diff
@@ -58,7 +58,7 @@
       <ul>
       <li>your salary is reviewed annually - most teachers get a pay rise every year, depending on your school. You could earn over £41k within 5 years</li>
       <li>you may get extra payments for taking on additional responsibilities, for example creating a timetable or becoming head of a department</li>
-      <li>as a headteacher you can expect to earn between £53k to £139k depending on your location and school</li>
+      <li>as a headteacher you can expect to earn between <%= v :salaries_headteacher_minshortened %> to <%= v :salaries_headteacher_maxshortened %> depending on your location and school</li>
```
Comment (Sarah-DfE) [inline app/views/content/landing/grow/_content.html.erb:61]:
```suggestion
      <li>as a headteacher you can expect to earn between <%= v :salaries_headteacher_minshortened %> and <%= v :salaries_headteacher_maxshortened %> depending on your location and school</li>
```

File: app/views/content/landing/how-much-do-teachers-get-paid/_content.html.erb [hunk 3]
```diff
@@ -9,7 +9,7 @@
 
       <ul>
         <li>teachers who make a sustained contribution to their school could expect to earn over £43k</li>
-        <li>headteachers are usually the highest paid people in a school and could expect to earn around £53k to £139k depending on the school and location</li>
+        <li>headteachers are usually the highest paid people in a school and could expect to earn around <%= v :salaries_headteacher_minshortened %> to <%= v :salaries_headteacher_maxshortened %> depending on the school and location</li>
```
Comment (Sarah-DfE) [inline app/views/content/landing/how-much-do-teachers-get-paid/_content.html.erb:12]:
```suggestion
        <li>headteachers are usually the highest paid people in a school and could expect to earn around <%= v :salaries_headteacher_minshortened %> and <%= v :salaries_headteacher_maxshortened %> depending on the school and location</li>
```

File: app/views/content/landing/what-teaching-taught-me/_content.html.erb [hunk 4]
```diff
@@ -51,7 +51,7 @@
       <ul>
         <li>your salary is reviewed annually - most teachers get a pay rise every year, depending on your school. You could earn over £41k within 5 years.</li>
         <li>you may get extra payments for taking on additional responsibilities, for example creating a timetable or becoming head of a department.</li>
-        <li>as a headteacher you can expect to earn between £53k to £139k depending on your location and school.</li>
+        <li>as a headteacher you can expect to earn between <%= v :salaries_headteacher_minshortened %> to <%= v :salaries_headteacher_maxshortened %> depending on your location and school.</li>
```
Comment (Sarah-DfE) [inline app/views/content/landing/what-teaching-taught-me/_content.html.erb:54]:
```suggestion
        <li>as a headteacher you can expect to earn between <%= v :salaries_headteacher_minshortened %> and <%= v :salaries_headteacher_maxshortened %> depending on your location and school.</li>
```

**Review by Sarah-DfE — COMMENTED** [review]
@gemmadallmandfe The variables all look good to me. I've changed the wording 'to' to 'and', as in 'between £X and £X'

**Review by Sarah-DfE — APPROVED** [review]
all good

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me

**Pull request merged**